backend/app/services/adapters/task_kinds/queries.py

```
@@ -0,0 +1,977 @@
+        tasks: List[TaskResource],
+        user_id: int,
+        member_counts: Dict[int, int],
+    ) -> List[Dict[str, Any]]:
+        """Build lightweight task list result."""
+        result = []
+        for task in tasks:
+            task_crd = Task.model_validate(task.json)
+
+            task_type = (
+                task_crd.metadata.labels
+                and task_crd.metadata.labels.get("taskType")
```

Ruff (0.14.13)

Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)